### ISSUE TITLE
support java 8 ?

### ISSUE THREAD

**Issue by OhadR** [stevesaliman/gradle-cobertura-plugin#95]
Hi,

I use version 2.2.7 of gradle-cobertura-plugin, which AFAIK supports cobertura version >2.1, that supports java 8.

However, when we have a code here with lambda expression, we get the following error:

java.lang.IllegalArgumentException: INVOKESPECIAL/STATIC on interfaces require ASM 5

Does it make sense?

Ohad

https://github.com/cobertura/cobertura/issues/166

**Comment by stevesaliman** [comment]
The plugin should run fine with Java 8, but Cobertura is still having problem with lambdas, as seen in https://github.com/cobertura/cobertura/issues/176.

I've been following that issue for a while now, but haven't heard of any fixes for it yet.

**Comment by Kalaiventhan** [comment]
we are facing lambda expression issue using 2.3.0 Gradle cobertura plugin and Java8. Is this issue fixed?

**Comment by eschizoid** [comment]
+1

**Comment by stevesaliman** [comment]
Unfortunately, it doesn't look like this has  been addressed by the Cobertura team yet.

**Comment by eschizoid** [comment]
Yeah that thread has been open for almost 1.5 years. At this point I don think the cobertura team has plans to fix it :disappointed:
